chess_api/chess_api/chess_api.py

```diff
@@ -0,0 +1,56 @@
+from flask import (
+    Blueprint, request, jsonify
+)
+from . import chess_logic
+
+bp = Blueprint('chess_api', __name__, url_prefix='/chess_api')
+
+@bp.route("/start-game", methods=["POST"])
+def initialize_board():
+    if request.method == "POST":
+        chess_logic.initialize_board()
+        try:
+            return "Chess engine set and board cleared."
+        except:
+            return "Could not determine intent"
```

Comment:
For all of the routes with `try... except`, these lines are not doing anything since they are just over return statements. `except` will catch any exceptions that are thrown within the try block. I would either 1) move the `chess_logic.initialize_board()` inside the try block or 2) remove the try block entirely. By default, flask will send a response with a non-200 response code for any errors that happen in a route. This will need to be done for all the routes.